tools-local/prebuilt-baseline-online.xml

```diff
@@ -264,6 +264,7 @@
     <Usage Id="System.Diagnostics.Tracing" Version="4.1.0" />
     <Usage Id="System.Diagnostics.Tracing" Version="4.3.0" />
     <Usage Id="System.Drawing.Common" Version="4.7.0" />
+    <Usage Id="System.Drawing.Common" Version="5.0.2" />
```

Comment:
Should this be removed after https://github.com/dotnet/source-build-reference-packages/pull/293 flows in?

Reply:
No.  This is the online baseline, so it will still exist there, since we don't use SBRP when building online.